src/Adapter/MockStorageAdapter.php

```diff
@@ -57,6 +62,8 @@ public function fetch(RequestInterface $request)
         foreach ($this->namingStrategies as $strategy) {
             if (file_exists($filename = $this->getFilename($strategy->filename($request)))) {
                 return Psr7\parse_response(file_get_contents($filename));
+            } else {
+                var_dump($filename);
```

Comment:
Isn't that a remaining debug statement?

Reply:
Sorry about that! Goes to prove there's no amount of experience that protects me from noob mistakes :)